### PR TITLE
docs(telemetry): Add Telemetry page

### PR DESCRIPTION
**Pull request by splitinfinities**
Also allows opting out of listing parent sections in the sidebar.

Internal ticket ID: STENCIL-35

### CODE REVIEW

File: src/assets/docs-structure.json [hunk 1]
```diff
@@ -400,5 +400,16 @@
         "filePath": "https://github.com/ionic-team/stencil"
       }
     ]
+  },
+  {
+    "text": "Legal",
+    "hideFromSidebar": true,
```
Comment (rwaskiewicz) [inline src/assets/docs-structure.json:406]: Is the idea to hide this until the feature goes live?

File: src/docs/telemetry.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+title: Telemetry
+description: Stencil Telemetry usage information
+url: /telemetry
+contributors:
+  - splitinfinities
+---
+
+# Stencil CLI telemetry
+
+Stencil collects anonymous telemetry data about usage of our command line interface. Participation in this program is optional, and [you may opt-out](#how-do-i-opt-out) if you'd prefer not to share any information.
```
Comment (rwaskiewicz) [inline src/docs/telemetry.md:11]: This is more a question for Josh - should we say "as of version 2.7.0...." here?

File: src/assets/docs-structure.json [hunk 3]
```diff
@@ -400,5 +400,16 @@
...
+    "children": [
+      {
+        "text": "Telemetry",
+        "filePath": "/assets/docs/telemetry.json",
+        "url": "/telemetry"
```
Comment (rwaskiewicz) [inline src/assets/docs-structure.json:411]: should this be 
```suggestion
        "url": "/docs/telemetry"
```

Comment (splitinfinities) [reply src/assets/docs-structure.json:411]: No, it should be /telemetry

File: src/assets/docs-structure.json (hunk 1 above)
Comment (rwaskiewicz) [reply src/assets/docs-structure.json:406]: Talked offline about this

File: src/docs/telemetry.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+We do not collect any metrics which may contain sensitive data. This includes, but is not limited to: environment variables, file paths, contents of files, logs, or serialized JavaScript errors.
+
+### Will This Data Be Shared?
+
+No. Data collected will be shared outside of Ionic. The data collected is anonymous, not traceable to the source, and only meaningful in aggregate form. We take your privacy and our security very seriously. Stencil telemetry falls under the [Ionic Privacy Policy](https://ionicframework.com/privacy).
+
+### How Do I Opt-Out?
+
+You may opt out-by running the following command in the root of your project directory:
```
Comment (rwaskiewicz) [inline src/docs/telemetry.md:61]: Should this be 
```suggestion
You may opt-out by running the following command in the root of your project directory:
```

**Review by rwaskiewicz — APPROVED** [review]
I had a few English nitpicks, but if Josh is happy I'm happy

File: src/docs/telemetry.md (hunk 2 above)
Comment (splitinfinities) [reply src/docs/telemetry.md]: Added

**Review by rwaskiewicz — APPROVED** [review]

**Pull request merged**